_data/events.yml

```diff
@@ -57,14 +57,17 @@
   content: "Hands-On"
   type: ["On-Site"]
   level: Introduction
-  duration: ""
-  video: ""
-  image: ""
-  linkedIn: ""
-  instagram: ""
-  twitter: ""
-  githubUrl: ""
-  blogPost: ""
```

Comment:
Can you add a link to the blog post:
https://itenium.be/blog/design/bootcamp-3d-modelling/